setup.py

```diff
@@ -35,7 +35,7 @@ def readme():
         "docopt >= 0.6.2",
         "mongo-db-from-config >= 0.0.1",
         "netaddr >= 0.7.10",
-        "pymongo >= 2.9.2, < 3",
+        "pymongo >= 3.7.2",
```

Comment:
Is it OK to install this version?  I'm unsure if [this comment](https://github.com/cisagov/cyhy_amis/blob/develop/ansible/roles/mongo/tasks/main.yml#L9-L18) is relevant here.

Reply:
That was changed to mirror the version mongo-db-from-config uses. I can revert, but on installing requirements it does throw a warning because of the difference.

Reply:
I suspect it's OK with 3.7.2.  What do you think, @dav3r?  As I recall the downgrade of pymongo was only necessary when using specific MongoDB features.

Reply:
I think 3.7.2 should be ok now that `cyhy-feeds` no longer requires `cyhy-core`. @mcdonnnj is going to test this before we deploy it to production, so we will know for sure if there's an issue.